cli/command/stack/deploy.go

```diff
@@ -520,6 +528,45 @@ func convertService(
 	return serviceSpec, nil
 }
 
+func convertHealthcheck(healthcheck *composetypes.HealthCheckConfig) (*container.HealthConfig, error) {
+	if healthcheck == nil {
+		return nil, nil
+	}
+	var (
+		err               error
+		test              []string
+		timeout, interval time.Duration
+		retries           int
+	)
+	// FIXME(vdemeester) decide what mean NONE for healthcheck
+	if len(healthcheck.Command) == 0 || healthcheck.Command[0] == "none" {
+		test = []string{"NONE"}
```

Comment:
Yes, this needs to be an error:

```yaml
healthcheck:
  disable: true
  command: foo
```

That config doesn't make any sense. It doesn't make any more time to comment out the `command` line.